_data/hosting_providers.json

```
@@ -2188,5 +2188,15 @@
     "plan": "https://www.easyname.com/en/hosting/free-lets-encrypt-ssl-certificate",
     "reviewed": "2019.7.10",
     "note": ""
+  }
+     {
+    "name": "Thexyz",
+    "link": "https://www.thexyz.com",
+    "category": "partial",
+    "tutorial": "https://www.thexyz.com/account/knowledgebase/117/How-do-I-install-a-Lets-Encrypt-SSL-certificate-on-my-site.html",
+    "announcement": "[No need to find this]",
+    "plan": "",
+    "reviewed": "",
```

Let's add today's date so we know when this was last updated.
```suggestion
    "reviewed": "2019.7.22",
```